prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+let number1 = 3;
+let number2 = 7;
+let product = number1 * number2;
+console.log('Value of product:', product);
+console.log('Type of produce:', typeof product);
+
+let charge = 80;
+let payment = 51;
+let amountRemaining = charge - payment;
+console.log('Value of amountRemaining:', amountRemaining);
+console.log('Type of amountRemaining:', typeof amountRemaining);
+
+let tests = 39;
+let assignments = 71;
+let final = 99;
+let grade = (tests + assignments + final) / 3;
+console.log('Value of grade:', grade);
+console.log('Type of grade:', typeof grade);
+
+let firstName = 'Andrew';
+let lastName = 'Owyang';
+let fullName = firstName + ' ' + lastName;
+console.log('Value of full Name:', fullName);
+console.log('Type of fullName:', typeof fullName);
+
+let pH = 3;
+let isAcidic = pH < 7;
+console.log('Value of isAcidic:', isAcidic);
+console.log('Type of isAcidic', typeof isAcidic);
+
+let numberOfSoldiers = 21;
+let isSparta = numberOfSoldiers === 300;
+console.log('Value of isSparta:', isSparta);
+console.log('Type of isSparta:', typeof isSparta);
+
+let nickname = fullName;
+nickName = nickname + '  is the GOAT';
```

Comment:
Close! The exercise asks that you use the '[addition assignment](https://lms.learningfuze.com/code-exercises/Learning-Fuze/curriculum/prep-javascript-operators-and-expressions?v=b9210fce4aa431881d42ca5f731d8f6775bb7db3#assignment-operators)' operator. 

It should look like:

`nickname += ' is the GOAT';`